haechan45th/src/services/userService.js

```diff
@@ -25,7 +35,23 @@ const signUp = async (name, email, password, profileImage) => {
     }
 };
 
+const signIn = async (email, password) => {
+    try {
+        const user = await userDao.getUserByEmail(email);
+        if (!user || !await bcrypt.compare(password, user.password)) {
+            throw new Error('KEY_ERROR');
+        }
+        const token = jwt.sign({ email: user.email }, config.JWT_SECRET_KEY, { expiresIn: "1h" });
```

Comment:
```suggestion
        const token = jwt.sign({ id: user.id }, config.JWT_SECRET_KEY, { expiresIn: "1h" });
```
토큰을 통해 인가과정을 거치는 거의 모든 api의 내부 로직에서 userId가 필요하므로, 이를 위해서는 email을 통해 만들기보다는 id를 통해 만들어주셔야합니다.